hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -112,8 +112,11 @@ public class BlockOutputStream extends OutputStream {
   private final AtomicReference<IOException> ioException;
   private final ExecutorService responseExecutor;
 
-  // the effective length of data flushed so far
-  private long totalDataFlushedLength;
+  // the effective length of data sent to datanodes (via writeChunk).
+  private long totalSyncedDataLength;
+
+  // The effective length of data flushed to datanodes (via putBlock).
+  private long totalFlushedDataLength;
```

Comment:
Not sure `Synced` and `Flushed` are the best modifier to differentiate both.

Maybe just call them

`totalWriteChunkDataLength`

`totalPutBlockDataLength`

?

Reply:
Sounds good. Updated.